src/pages/Lists/Lists.vue

```diff
@@ -24,9 +24,20 @@
             :options="{group:{ name:'lists'}}"
             @end="end"
         >
-            <div v-for="({name, games}, index) in lists" :key="name" class="list" >
+            <div v-for="({name, games}, index) in lists" :key="name" class="list">
                 <div class="list-header">
-                    <strong>{{name}}</strong>
+                    <strong
+                        v-show="!editing || editingIndex != index"
```

Comment:
`!==` is prefered

Reply:
Ugh, forgot Javascript equality is so much fun to work with :P